collada_urdf/src/urdf_to_collada.cpp

```diff
@@ -50,13 +43,11 @@ int main(int argc, char** argv)
         return -1;
     }
 
-    ros::init(argc, argv, "urdf_to_collada");
-
     std::string input_filename(argv[1]);
     std::string output_filename(argv[2]);
 
     urdf::Model robot_model;
-    if( !robot_model.initFile(input_filename) ) {
+    if (!robot_model.initFile(input_filename)) {
         ROS_ERROR("failed to open urdf file %s", input_filename.c_str());
```

Comment:
Might need to replace with `std::cerr` or `#include <ros/rosconsole.h>`.

Reply:
Yeah, good point.  It compiled before, but I'm not sure why.  I've changed this to a `std::cerr` now, since this is a standalone program.